### PR TITLE
Clicontext

### PR DESCRIPTION
**Pull request by baude**
create_cli_test.go: Unittests
    
Create unittests for the create_cli helper functions. As
such, remove cli context usage from the functions to
further divide between cli input and parsing.  Also, simplifies
unit testing.

vendor in assert package

### CODE REVIEW

**Comment by mheon** [comment]
LGTM, though I'm hesitant to merge before we get PR testing working

**Comment by rhatdan** [comment]
Needs a rebase. I believe.

**Comment by baude** [comment]
if CI passes, please review and/or merge @rhatdan

File: libpod/util_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+)
+
+func TestStringInSlice(t *testing.T) {
+	// string is in the slice
+	assert.True(t, StringInSlice("one", sliceData))
+	// string is not in the slice
+	assert.False(t, StringInSlice("five", sliceData))
+	// string is not in empty slice
+	assert.False(t, StringInSlice("one", []string{}))
+}
```
Comment (rhatdan) [inline libpod/util_test.go]: Add line

**Comment by rhatdan** [comment]
One Nit, Fix this and then we can merge.


LGTM

File: libpod/util_test.go (hunk 1 above)
Comment (baude) [reply libpod/util_test.go]: @rhatdan are you wanting a newline? if so, gofmt keeps yanking it out.

**Pull request merged**